### ISSUE TITLE
Unfolded image should be converted to float

### ISSUE THREAD

**Issue by jccaicedo** [cytomining/DeepProfiler#254]
The normalization operation that follows image folding stores de result as `int8`, resulting in loss of information.

https://github.com/cytomining/DeepProfiler/blob/3081d667d44b1c2dcd87263cf2cdbafe4b99ca71/deepprofiler/imaging/cropping.py#L44

Add `.astype(np.float)` at the end of line 44

**Comment by Arkkienkeli** [comment]
@jccaicedo Do we need to redo all experiments with compressed images?

**Comment by Arkkienkeli** [comment]
Training experiment old version vs new (`*_re`)

https://www.comet.ml/arkkienkeli/ta-orf/9ffcbdf2b0a642a692152316510f8eb6/a449aea6c4d24ed5b46b0b77c8c9e016/compare?experiment-tab=chart&showOutliers=true&smoothing=0&transformY=smoothing&xAxis=step

**Comment by Arkkienkeli** [comment]
https://github.com/cytomining/DeepProfiler/commit/8d494c5b307a721f456452c82ceb5a52283e5d82

**Comment by jccaicedo** [comment]
So in conclusion, the fix produces slightly lower validation loss and slightly higher validation accuracy. Also, in the downstream analysis we observed improved performance. The plot below shows the folds of enrichment for two experiments in the TAORF dataset, each with the same configuration (EfficientNet): one before the fix (old in green) and another after the fix (replicate in blue). The fix basically adjusts numeric precision of normalized pixels, and it has an impact in performance.

![image](https://user-images.githubusercontent.com/7527135/107383049-97520f00-6abe-11eb-8e40-4e2c4223db49.png)
